README.rst

```diff
@@ -124,6 +124,58 @@ implemented:
 
 * ``core.routing.olsr`` provides information about the node's OLSR routing daemon.
 
+Development setup
+-----------------
+
+To build the master version of nodewatcher agent package a buildroot needs to be set up. Checkout `firmware core`_ and use the scripts which automatically download all the relevant feeds and create the buildroot in ``build/lede``::
+
+  ./lede/scripts/prepare v17.01.2 source.git 2da512ecf4631cd7812283f0931cf6bbf842a313
```

Comment:
Perhaps you should add a comment that this is just an example version/commit and the actual commit may change and should be looked up in the repository.